bin/fuel-core/src/cli/run.rs

```diff
@@ -189,12 +189,13 @@ pub struct Command {
     #[arg(long = "native-executor-version", env)]
     pub native_executor_version: Option<StateTransitionBytecodeVersion>,
 
-    /// The starting gas price for the network
-    #[arg(long = "starting-exec-gas-price", default_value = "0", env)]
-    pub starting_exec_gas_price: u64,
+    /// The starting execution gas price for the network
+    #[arg(long = "starting-gas-price", default_value = "0", env)]
+    pub starting_gas_price: u64,
 
     /// The percentage change in gas price per block
-    #[arg(long = "gas-price-change-percent", default_value = "0", env)]
+    #[arg(long = "gas-price-change-percent", default_value = "2", env)]
+    // https://docs.rs/tokio/latest/tokio/time/fn.pause.html
```

Comment:
a wild comment appeared?

Reply:
Oh don't know how that got there :sweat_smile:

Reply:
Addressed in https://github.com/FuelLabs/fuel-core/pull/2507/commits/3571bba55e3d260981b27063b4f36ad5168b02b1